native/Avalonia.Native/inc/avalonia-native.h

```diff
@@ -188,11 +189,11 @@ AVNCOM(IAvaloniaNativeFactory, 01) : IUnknown
     virtual HRESULT CreateClipboard(IAvnClipboard** ppv) = 0;
     virtual HRESULT CreateCursorFactory(IAvnCursorFactory** ppv) = 0;
     virtual HRESULT ObtainGlDisplay(IAvnGlDisplay** ppv) = 0;
-    virtual HRESULT ObtainAppMenu(IAvnAppMenu** retOut) = 0;
-    virtual HRESULT SetAppMenu(IAvnAppMenu* menu) = 0;
-    virtual HRESULT CreateMenu (IAvnAppMenu** ppv) = 0;
-    virtual HRESULT CreateMenuItem (IAvnAppMenuItem** ppv) = 0;
-    virtual HRESULT CreateMenuItemSeperator (IAvnAppMenuItem** ppv) = 0;
+    virtual HRESULT ObtainAppMenu(IAvnMenu** retOut) = 0;
```

Comment:
Do we still need both of them? We should probably always create an instance via `CreateMenu` and then set it via `SetAppMenu`

Reply:
removed the obtain methods.